README.md

```diff
@@ -8,6 +8,10 @@ pylabrad is an interface to the LabRAD system in python with support for both cl
 For general information about the LabRAD system/protocol see [the labrad repository](https://github.com/labrad/labrad) and associated [wiki](https://github.com/labrad/labrad/wiki).
 For help getting started and understanding pylabrad take a look at the [wiki](https://github.com/labrad/pylabrad/wiki).
 
+In addition to the basic labrad client/server support, this package also includes a tool called the "node" server in the `labrad.node` package.
+This server allows you to manage starting and stopping other servers on a remote machine, which can be useful in distributed setups.
```

Comment:
Right now this seems to say that starting/stopping other servers on a machine other than this one with pylabrad. A clearer way to say this would be "This server allows you to use a remote machine to manage starting and stopping other servers, which..."